code/index.html

```diff
@@ -1,22 +1,114 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
 
-    <section class="">
-      <!-- The other news  -->
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Fooder Magazine - Keep up with the latest of the foodie community</title>
+  <link rel="stylesheet" href="style.css">
+  <!--<link rel="stylesheet" href="./style.css" />-->
+  <!-- dont forget the meta tag for responsiveness -->
+</head>
+
+<body>
+
+  <header>
+    <img src="LogoFooderWithText.png" alt="Fooder Logo">
+    <img class="logo" src="LogoFooder.png" alt="Fooder Logo">
+    <div class="container">
+      <h1>Fooder Magazine</h1>
+      <h2>Get the latest bites of news from the foodie community</h2>
+    </div>
+  </header>
+
+  <nav>
+    <div class="hamburger">
+      <div class="line"></div><!-- 
+      <div class="line"></div>
+      <div class="line"></div> -->
+    </div>
+    <ul class="nav-links">
+      <li class="link"><a href="#">Home</a></li>
+      <li class="link"><a href="#">Archive</a></li>
+      <li class="link"><a href="#">Contact</a></li>
+    </ul>
+  </nav>
+
+
+  <main>
+    <article class="big-news">
+      <img src="pictures/bigkitchen.jpg" alt="Restaurant kitchen">
+      <div class="big-news-text">
+        <h2>Jennies fusion-corner big success</h2>
+        <p>Jennie's got a long experience in food and we can tell. This new little fusion-corner of the city is the place to be. We sent our Fooder troop to try out her new dishes and they were not dissapointed. Lorem ipsum
+          dolor sit amet consectetur adipiscing elit nunc, lectus vestibulum aenean a montes commodo vitae,
+          taciti dictum maecenas rhoncus diam convallis sollicitudin. Malesuada ultrices eget diam ridiculus facilisis
+          faucibus at tincidunt mollis, ad libero vivamus ante pretium laoreet montes odio vestibulum, mi cubilia augue
+          placerat eros fringilla velit vel. </p>
+      </div>
+    </article>
+
+
+    <section class="card-container">
+      <div class="article-card one"><img src="pictures/salmon.jpg" alt="">
+        <div class="card-text">
+          <h3>Olszewski opens up new restaurant</h3>
+          <p>Maksymilian has done it once again. With his new restaurant he experiments with traditional cooking-methods
+            but with a twist to create dishes beyond our imagination.</p>
+        </div>
+      </div>
+      <div class="article-card two"><img src="pictures/cookingpasta.jpg" alt="">
```

Comment:
If the cards are not styled differently you don't need to give them different class names :)